lua/neogit/popups/tag/init.lua

```diff
@@ -0,0 +1,29 @@
+local actions = require("neogit.popups.tag.actions")
+local popup = require("neogit.lib.popup")
+
+local M = {}
+
+local noop = function() end
+
+function M.create()
+  local p = popup
+    .builder()
+    :name("NeogitTagPopup")
+    :arg_heading("Arguments")
+    :switch("f", "force", "Force")
+    :switch("a", "annotate", "Annotate")
+    :switch("s", "sign", "Sign")
+    :switch("u", "local-user", "Sign as", { enabled = false })
+    :new_action_group("Create")
+    :action("t", "tag", actions.create_tag)
+    :action("r", "release", noop)
```

Comment:
You don't need the `noop` function, if the last argument is nil then the action show up as disabled on the popup.